src/SignalR/samples/SignalRSamples/Hubs/Streaming.cs

```diff
@@ -12,16 +12,16 @@ namespace SignalRSamples.Hubs
 {
     public class Streaming : Hub
     {
-        public async IAsyncEnumerable<int> AsyncEnumerableCounter(int count, int delay)
+        public async IAsyncEnumerable<int> AsyncEnumerableCounter(int count, double delay)
         {
             for (var i = 0; i < count; i++)
             {
                 yield return i;
-                await Task.Delay(delay);
+                await Task.Delay((int)delay);
```

Comment:
Can we add a skipped test for System.Text.Json casting double to int automatically before merging?

Reply:
I'm not sure we want this behavior. But I'll add a commented out test, and we can decide later.